test/test_sim_utils.py

```diff
@@ -454,3 +449,75 @@ def test_region_containment_utils():
         assert livingroom_ratio > 0
         assert livingroom_ratio < 0.51
         assert bedroom_ratio > 0.51
+
+
+@pytest.mark.skipif(
+    not built_with_bullet,
+    reason="Raycasting API requires Bullet physics.",
+)
+@pytest.mark.skipif(
+    not osp.exists("data/replica_cad/"),
+    reason="Requires ReplicaCAD dataset.",
+)
+def test_ao_open_close_queries():
+    sim_settings = default_sim_settings.copy()
+    sim_settings[
+        "scene_dataset_config_file"
+    ] = "data/replica_cad/replicaCAD.scene_dataset_config.json"
+    sim_settings["scene"] = "apt_0"
+    hab_cfg = make_cfg(sim_settings)
+    with Simulator(hab_cfg) as sim:
+        # for revolute hinge doors
+        fridge = sutils.get_obj_from_handle(sim, "fridge_:0000")
+
+        # for prismatic drawers
+        kitchen_counter = sutils.get_obj_from_handle(
+            sim, "kitchen_counter_:0000"
+        )
+
+        # for prismatic (sliding) doors
+        cabinet = sutils.get_obj_from_handle(sim, "cabinet_:0000")
+
+        objects = [fridge, kitchen_counter, cabinet]
+        for obj in objects:
+            for link_id in obj.get_link_ids():
+                if obj.get_link_joint_type(link_id) in [
+                    JointType.Revolute,
+                    JointType.Prismatic,
+                ]:
+                    if (
+                        "cabinet" in obj.handle
+                        and "right_door" in obj.get_link_name(link_id)
+                    ):
+                        print(
+                            "TODO: Skipping 'cabinet's right_door' link because it does not follow conventions. Axis should be inverted."
```

Comment:
not sure if I understand this printed warning. Is this specific to this one example?

Reply:
Yeah, this is related to a specific asset which does not agree with our API. As mentioned in the "limitations" section of the PR description.